crp/concepts.py

```diff
@@ -100,11 +100,10 @@ def reference_sampling(self, relevance, layer_name: str = None, max_target: str
 
         # channel maximization target
         if max_target == "sum":
-
-            rel_l = torch.sum(relevance.view(*relevance.shape[:2], -1), dim=-1)
+            rel_l = torch.sum(rel_l, dim=-1)
```

Comment:
At the first glance, it makes sense to simplify the expression. Good catch! But here I made it more complex on purpose, so that we can cover not only Conv2d but also Linear and ConNd layer.